epilepsy12/common_view_functions/recalculate_form_generate_response.py

```diff
@@ -519,7 +520,7 @@ def expected_score_for_single_episode(episode):
     # essential fields already accounted for are:
     # seizure_onset_date
     # seizure_onset_date_confidence
-    # episode_definition
+    # episode_definition - DEPRECATED
```

Comment:
Reading the thread in #1015 I think it's more optional than deprecated?

Reply:
yes good point.